File: Master/Year 1/Semester 1/AMSS/Proiect/app/src/main/java/ro/robertto/rescuepets/presentation/activities/GuidePage1.java
package ro.robertto.rescuepets.presentation.activities;

import android.content.Intent;
import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import ro.robertto.rescuepets.databinding.FirstPageGuideBinding;

public class GuidePage1 extends AppCompatActivity {
    private FirstPageGuideBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = FirstPageGuideBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        binding.SecondPageButton.setOnClickListener(view -> {
            Intent intent = new Intent(this, GuidePage2.class);
            startActivity(intent);
        });

        binding.SecondPageButtonSkip.setOnClickListener(view -> {
            Intent intent = new Intent(this, LogInActivity.class);
            startActivity(intent);
        });
    }
}

File: Anul 3/Semestrul 2/Android/Proiect/EmergencyApp/app/src/main/java/com/lifeSavers/emergencyapp/activities/GuidePage1.kt
package com.lifeSavers.emergencyapp.activities

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.lifeSavers.emergencyapp.databinding.FirstPageGuideBinding

class GuidePage1 : AppCompatActivity() {
    private lateinit var binding: FirstPageGuideBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = FirstPageGuideBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.SecondPageButton.setOnClickListener {
            val intent = Intent(this, GuidePage2::class.java)
            startActivity(intent)
        }

        binding.SecondPageButtonSkip.setOnClickListener {
            val intent = Intent(this, LogInActivity::class.java)
            startActivity(intent)
        }
    }
}
